POGOLib.Official/Net/Session.cs

```diff
@@ -149,7 +144,9 @@ public void Shutdown()
             var result = Configuration.Hasher.PokemonVersion.CompareTo(pogoVersion);
             if (result != 0)
             {
-                throw new HashVersionMismatchException($"The version of the {nameof(Configuration.Hasher)} ({Configuration.Hasher.PokemonVersion}) does not match the minimal API version of PokemonGo ({pogoVersion}). Set 'Configuration.IgnoreHashVersion' to true if you want to disable the version check.");
+                var str1 = "Set 'Configuration.IgnoreHashVersion' to true if you want to disable the version check.";
+                var str2 = $"The version of the {nameof(Configuration.Hasher)} ({Configuration.Hasher.PokemonVersion}) does not match the minimal API version of PokemonGo ({pogoVersion}). {str1}";
+                throw new HashVersionMismatchException(str2);
```

Comment:
that was not importan change, some, text editors seeing that the lenght of that string is greater than allow by them.